src/App.tsx

```diff
@@ -24,6 +28,10 @@ function App() {
                     <Switch>
                         <Route exact={true} path={'/'} component={Forside} />
                         <Route exact={true} path={'/helse'} component={Helse} />
+                        <Route exact={true} path={'/steg1'} component={Steg1} />
+                        <Route exact={true} path={'/steg2'} component={Steg2} />
+                        <Route exact={true} path={'/steg3'} component={Steg3} />
+                        <Route exact={true} path={'/steg4'} component={Steg4} />
```

Comment:
Disse kan nå rendres inne i Side komponenten. Siden vi har/skal fjerne forside kunne vi kanskje kalt den Steg istedet for Side :)

Reply:
`Side` hos oss er definert slik at hvert Steg rendrer en Side, så Side er på en måte en hjelpekomponent. Den kan godt hete `Steg`

Jeg tenker at vi kan ha en component som heter (foreløpig)`Søknad` som har en Switch til alle stegene, og at stegene rendrer hver sin Side/Steg komponent. Det ligger hvertfall nærmere den løsningen vi har nå.

Vi tester ut dette, og hvis det er helt dust så kan vi gå tilbake til dette forslaget. Ok? 👍